### PR TITLE
Get latest

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[439618c](https://github.com/OfficeDev/office-js-docs-reference/pull/438/commits/439618c94651ed9a2b530093a6cef44b8f768bbd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/office.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.binding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.binding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.context.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.context.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.dialogoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.dialogoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.document.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.document.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.filetype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.filetype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.officetheme.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.officetheme.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.platformtype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.platformtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.settings.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.settings.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.settingschangedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.settingschangedeventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.tablebinding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.tablebinding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.customproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.customproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.diagnostics.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.diagnostics.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.item.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.item.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.itemcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.itemcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.itemread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.itemread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.mailbox.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.mailbox.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.mailboxenums.owaview.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.mailboxenums.owaview.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.userprofile.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.userprofile.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.customproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.customproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.diagnostics.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.diagnostics.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.itemcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.itemcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.itemread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.itemread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.mailbox.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.mailbox.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.owaview.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.owaview.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.customproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.customproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.diagnostics.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.diagnostics.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.itemcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.itemcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.itemread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.itemread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.mailbox.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.mailbox.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.mailboxenums.owaview.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.mailboxenums.owaview.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.customproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.customproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.diagnostics.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.diagnostics.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.itemcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.itemcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.itemread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.itemread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.mailbox.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.mailbox.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C26%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201906261818522810-438%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=dcOvia5HAOn%2BIHS6DjQXn0sSPeXMCwx5Vfbly%2BSMGIQ%3D&st=2019-06-26T18%3A19%3A58Z&se=2019-07-27T18%3A24%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bad785e](https://github.com/OfficeDev/office-js-docs-reference/pull/438/commits/bad785ee88cad5bfdc649e80b94f21147dce740e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[README.md](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/README.md) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.binding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.binding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.context.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.context.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.dialogoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.dialogoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.document.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.document.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.filetype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.filetype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.officetheme.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.officetheme.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.platformtype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.platformtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.settings.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.settings.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.settingschangedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.settingschangedeventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.tablebinding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/office/office.tablebinding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.customproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.customproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.diagnostics.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.diagnostics.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.item.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.item.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.itemcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.itemcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.itemread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.itemread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.mailbox.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.mailbox.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.mailboxenums.owaview.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.mailboxenums.owaview.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.userprofile.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.userprofile.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.customproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.customproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.diagnostics.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.diagnostics.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.itemcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.itemcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.itemread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.itemread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.mailbox.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.mailbox.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.owaview.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.owaview.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.customproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.customproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.diagnostics.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.diagnostics.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.itemcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.itemcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.itemread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.itemread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.mailbox.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.mailbox.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.mailboxenums.owaview.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.mailboxenums.owaview.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_2/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.customproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.customproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.diagnostics.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.diagnostics.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.itemcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.itemcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.itemread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_3/office.itemread.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C26%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201906261832345430-438%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=5DMvfH7nYlOXYLdpQ5%2F0Dz%2FtGvp4012%2BirAs%2FSNXARk%3D&st=2019-06-26T18%3A33%3A38Z&se=2019-07-27T18%3A38%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/docs-ref-autogen/outlook/office.messageread.yml [hunk 1]
```diff
@@ -1853,8 +1853,9 @@ items:
       Gets the id of the series that an instance belongs to.
 
 
-      In OWA and Outlook, the seriesId returns the Exchange Web Services (EWS) ID of the parent (series) item that this
-      item belongs to. However, in iOS and Android, the seriesId returns the REST ID of the parent item.
+      In Outlook on the web and desktop clients, the seriesId returns the Exchange Web Services (EWS) ID of the parent
+      (series) item that this item belongs to. However, in iOS and Android, the seriesId returns the REST ID of the
```
Comment (mscharlock) [inline docs/docs-ref-autogen/outlook/office.messageread.yml]:
```suggestion
      (series) item that this item belongs to. However, on iOS and Android, the seriesId returns the REST ID of the
```

File: docs/docs-ref-autogen/office/office.platformtype.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ items:
       - typeScript
     type: field
   - uid: office.Office.PlatformType.OfficeOnline
-    summary: The platform is Office Online.
+    summary: The platform is Office on the web (in a browser).
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/office/office.platformtype.yml]: If I understand it correctly, then no. I believe "Online" version targeted the browser. Then supported browsers on supported platforms should run this version.

File: docs/docs-ref-autogen/office/office.context.yml [hunk 3]
```diff
@@ -46,12 +46,12 @@ items:
...
 
 
-      `commerceAllowed` is only supported on Office for iPad.
+      `commerceAllowed` is only supported in Office on iPad.
 
 
       The iOS App Store doesn't support apps with add-ins that provide links to additional payment systems. However,
-      Office Add-ins running on the Windows desktop or for Office Online in the browser do allow such links. If you want
-      the UI of your add-in to provide a link to an external payment system on platforms other than iOS, you can use the
+      Office Add-ins running in Office on the Windows desktop or in the browser do allow such links. If you want the UI
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/office/office.context.yml]: I know, it kinda does. I thought about changing it to add-ins running in Office but then again there are a variety of add-in types...(sigh)

File: docs/docs-ref-autogen/outlook/office.appointmentcompose.yml [hunk 4]
```diff
@@ -509,8 +509,8 @@ items:
       session.
 
 
-      If your Office add-in is running in Outlook Web App, the addItemAttachmentAsync method can attach items to items
-      other than the item that you are editing; however, this is not supported and is not recommended.
+      If your Office add-in is running in Outlook on the web, the addItemAttachmentAsync method can attach items to
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/outlook/office.appointmentcompose.yml]: No as that got deprecated some time ago.

File: docs/docs-ref-autogen/outlook/office.appointmentcompose.yml [hunk 5]
```diff
@@ -1615,11 +1615,11 @@ items:
...
             Optional. An object literal that contains one or more of the following properties. asyncContext: Developers
             can provide any object they wish to access in the callback method. coercionType: If text, the current style
-            is applied in Outlook Web App and Outlook. If the field is an HTML editor, only the text data is inserted,
-            even if the data is HTML. If html and the field supports HTML (the subject doesn't), the current style is
-            applied in Outlook Web App and the default style is applied in Outlook. If the field is a text field, an
-            InvalidDataFormat error is returned. If coercionType is not set, the result depends on the field: if the
-            field is HTML then HTML is used; if the field is text, then plain text is used.
+            is applied in Outlook on the web and Windows. If the field is an HTML editor, only the text data is
+            inserted, even if the data is HTML. If html and the field supports HTML (the subject doesn't), the current
+            style is applied in Outlook on the web and the default style is applied in Outlook on desktop clients. If
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/outlook/office.appointmentcompose.yml]: I'm not sure. I was thinking of it as tablet, mobile device.

File: docs/docs-ref-autogen/outlook/office.appointmentread.yml [hunk 6]
```diff
@@ -1853,8 +1853,9 @@ items:
       Gets the id of the series that an instance belongs to.
 
 
-      In OWA and Outlook, the seriesId returns the Exchange Web Services (EWS) ID of the parent (series) item that this
-      item belongs to. However, in iOS and Android, the seriesId returns the REST ID of the parent item.
+      In Outlook on the web and desktop clients, the seriesId returns the Exchange Web Services (EWS) ID of the parent
+      (series) item that this item belongs to. However, in iOS and Android, the seriesId returns the REST ID of the
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/outlook/office.appointmentread.yml]: I'll make these suggestions in a subsequent DT PR.

**Pull request merged**